content/docs/1.7.0/snapshots-and-backups/backup-and-restore/create-a-backup.md

```diff
@@ -3,10 +3,12 @@ title: Create a Backup
 weight: 2
 ---
 
+## Incremental Backup
 Backups in Longhorn are objects in an off-cluster backupstore. A backup of a snapshot is copied to the backupstore, and the endpoint to access the backupstore is the backup target. For more information, see [this section.](../../../concepts/#31-how-backups-work)
 
 > **Prerequisite:** A backup target must be set up. For more information, see [Set the BackupTarget](../set-backup-target). If the BackupTarget has not been set, you'll be presented with an error.
 
+### Create an Incremental Backup Through UI
```

Comment:
nit: Just unify the wording style. We seem using `using` more.
```suggestion
### Create an Incremental Backup using UI
```

Reply:
fixed.